Sources/MenuAnimator.swift

```diff
@@ -129,15 +129,7 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     var present: Bool = false
     var interactionInProgress: Bool = false
 
-    private var transitionDuration: TimeInterval
-    private var visibleContentWidth: CGFloat
-    private var scaleDiff: CGFloat
-    private var useFinishingSpringOption: Bool
-    private var useCancellingSpringOption: Bool
-    private var finishingSpringOption: SpringOption
-    private var cancelingSpringOption: SpringOption
-    private var animationOptions: UIViewAnimationOptions
-    
+    var params: TransitionOptions = TransitionOptions()
```

Comment:
actually, I'd prefer to see `options`/`transitionOptions` instead of params. I think it more readable.